Documentation/KnownIssues.md

```diff
@@ -155,13 +155,13 @@ or by adding the property `<CopyLocalLockFileAssemblies>` to the project file
 
 NB. This **DOESN'T ALWAYS WORK**, for example in case of the shared framework <https://github.com/dotnet/cli/issues/12705#issuecomment-536686785>
 
-We can do nothing at the moment as this is a build behaviour out of our control.
```

Comment:
I have submitted https://github.com/coverlet-coverage/coverlet/pull/1769 to fix this separately.